payroll-processor-client/src/app/payroll/payroll-charts/payroll-charts.component.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  Component,
+  OnInit,
+} from '@angular/core';
+
+import { DataService } from 'src/app/data/data-service';
+import { NameValue } from 'src/app/data/state/data-model';
+
+@Component({
+  selector: 'app-payroll-charts',
+  templateUrl: './payroll-charts.component.html',
+  styleUrls: ['./payroll-charts.component.scss'],
+})
+export class PayrollChartsComponent implements OnInit {
+  departmentDistributionData: NameValue[] = [];
```

Comment:
When you make the changes to add the presenter service, mark these as `readonly` since you'll be assigning the values in this component's constructor.